src/components/modal/README.md

```diff
@@ -439,10 +439,13 @@ prop to `true`. Set it to `false` to re-enable both buttons.
 ## Multiple modal support
 Unlike native Bootstrap V4, Bootstrap-Vue supports multiple modals opened at the same time.
 
+Setting `stackable` to `false` on a specific modal will disable stacking for this specific modal
```

Comment:
Thinking we should use a `no-stacking` prop name (as there are testing/coding issues with dealing with props that default to true)

It would default to `false`, and if present on the modal:

```html
 <b-modal no-stacking ... >
```

Would disable the stacking mode without having to use the `v-bind:stackable="false"` (or shortcut of `:stackable="false"`).  Many people forget to add the `v-bind:` (or `:`) to props like this, and if thy do, then a string value becomes "truthy" and doesn't do what they expected.

Reply:
Otherwise, all looks good. Will play with this later today.